src/inspector/parsers/cbor/parser.ts

```diff
@@ -0,0 +1,45 @@
+import { Result, firstOk } from '../../calculatedAttributes/errors';
+import cbor from 'cbor';
+import { anyToByteArray } from './toByteArray';
+import { CBOR } from './cborSchema';
+
+export type ParsedCBOR = {
+  type: 'CBOR';
+  byteType: 'hex' | 'base64' | 'base64url';
+  payload: CBOR;
+};
+
+export function safeCBORParse(credential: string): Result<ParsedCBOR> {
+  const decoded = firstOk(anyToByteArray(credential));
+
+  if (decoded.kind === 'error') {
+    return {
+      kind: 'error',
+      error: {
+        name: 'No byteformat detected',
+        message: 'Not one of the supported byte formats (hex, base64url, base64)',
+      },
+    };
+  }
+
+  let cborDecoded = cbor.decodeFirstSync(decoded.value.byteArray, {
+    max_depth: 500,
+    tags: { 24: (x: Uint8Array) => cbor.decodeFirstSync(x) },
+  });
```

Comment:
This call here fails when sending for example `asd`. Not all strings fail. Only those that begin as valid cbor format. 
![image](https://github.com/thomsen85/vc-inspector/assets/70779496/8bb54659-5e91-43b1-9464-135605e8ed88)